### ISSUE TITLE
Possible regression on 0.8.1

### ISSUE THREAD

**Issue by tommywalkie** [pmndrs/valtio#110]
Tried using the newly released `0.8.1` version (via ES Modules) today and couldn't import any of Valtio methods.

[JSFiddle](https://jsfiddle.net/tommywalkie/vbn4kx3u/)

```js
// 0.8.0
import { subscribe as subscribeA } from 'https://cdn.skypack.dev/valtio@0.8.0'
// 0.8.1
import { subscribe as subscribeB } from 'https://cdn.skypack.dev/valtio@0.8.1'

console.log(subscribeA) // ✔️ works
console.log(subscribeB) // ❌ SyntaxError: import not found: subscribe
```

**Comment by dai-shi** [comment]
Thanks for reporting! Sorry for the inconvenience.
Okay, I need to look for some other solutions that work with esm too. 🤔

**Comment by dai-shi** [comment]
```js
import { subscribe as subscribeB } from 'https://cdn.skypack.dev/valtio@0.8.1/vanilla'
```
This seems to work. just FYI.

---

https://github.com/pmndrs/valtio/blob/a396bfdfde33a1ce93c1e9c24e387b068df3d50e/src/index.ts#L1

This doesn't seem to work with pika.

**Comment by tommywalkie** [comment]
@dai-shi That's possible. Looking at https://github.com/snowpackjs/snowpack/issues/2774, the `zarr` package seems to run with similar issues.

**Comment by dai-shi** [comment]
It turns out #112 didn't fix this issue at all.
https://jsfiddle.net/gpLesnzu/

Does anyone know how to work around this?

**Comment by lauritzsh** [comment]
I wanted to try this library since it seems simple and flexible, while allowing writing to state outside React compared to Jotai. However, using Snowpack for development, I have a problem importing just following the examples posted. With

```react
import { proxy } from 'valtio';

export const test = proxy({ hello: 'world' });
```

I get `Uncaught SyntaxError: The requested module '../../../../_snowpack/pkg/valtio.js' does not provide an export named 'proxy'`. Some workarounds seems to be `import { proxy } from 'valtio/vanilla'` but that results in `Uncaught Error: Please use proxy object`. The only thing working for me is 

```react
import v from 'valtio';

export const test = v.proxy({ hello: 'world' });
```

If I do `import * as v from 'valtio';` then the structure is

![image](https://user-images.githubusercontent.com/1979410/114417788-2f996c80-9bb2-11eb-8ae4-7adda7df1a12.png)

if that helps.

Just curious if it's related to this issue?

**Comment by dai-shi** [comment]
Oh, no. We would definitely like to fix the issue with snowpack.
The issue seems related to this one, but not 100% as the workaround is different.
I will investigate it when I have a chance to learn snowpack, but if someone can help on this, it'd be very appreciated.

**Comment by LXSMNSYC** [comment]
<img width="351" alt="Screen Shot 2021-07-09 at 10 35 15 PM" src="https://user-images.githubusercontent.com/4783372/125094636-03847d80-e106-11eb-9f30-0810c128e01f.png">

@dai-shi This should be fixed now in 1.0.7 (logged module object is from Skypack, Valtio 1.0.7)

**Comment by dai-shi** [comment]
Cool. Thanks for confirming.
